server/routes/index.ts

```diff
@@ -21,11 +23,17 @@ export default function routes(service: Services): Router {
   commonRoutes()
 
   get('/prisoner/:prisonerNumber', async (req, res, next) => {
-    const pageService = new PageService()
-    const prisonApi = new PrisonApiRestClient(res.locals.user.token)
+    const prisonerSearchClient = new PrisonerSearchClient(res.locals.clientToken)
+    const prisonerData: Prisoner = await prisonerSearchClient.getPrisonerDetails(req.params.prisonerNumber)
+
+    const prisonApi = new PrisonApiRestClient(res.locals.clientToken)
     const overviewPageService = new OverviewPageService(prisonApi)
-    const overviewPage = await overviewPageService.get(req.params.prisonerNumber)
-    pageService.renderPage<OverviewPage>(res, req.params.prisonerNumber, 'pages/index', overviewPage)
+    const overviewPageData = await overviewPageService.get(prisonerData)
```

Comment:
@davidmiddletonmoj, is this suppose to be prisoner Data or req.params.prisonerNumber?

Reply:
It's `prisonerData` now - the method pulls out a few bits of info from this object